source/jobs/JobsFeature.cpp

```diff
@@ -93,6 +93,7 @@ void JobsFeature::ackSubscribeToUpdateJobExecutionAccepted(int ioError)
         LOG_ERROR(TAG, errorMessage.c_str());
```

Comment:
We might wanna update the error message here and in other call back methods since it is same everywhere. (Line # 46, 63, 75, 92). User will not be able to debug which subscription call failed.

Reply:
That's a good suggestion, I'll make the update